oak_remote_attestation_verification/proto/v1/verification.proto

```diff
@@ -20,16 +20,16 @@ package oak.verification.v1;
 
 // Client-side options for verifying the attestation endorsement for a binary.
 message AttestationVerificationOptions {
-    // TEE platforms that the client is allowed to interact with, and can verify the evidence
-    // generated by them.
-    repeated TeePlatform supported_tee_platforms = 1;
-    // Whether to skip TEE certificate verification.
-    bool skip_tee_certificate_verification = 2;
-    // Verification option for each DICE layer.
-    map<string, LayerVerificationOptions> layer_verification_options = 3;
-    // If provided, will be used for verifying any layer that does not have a matching verification
-    // option in `layer_verification_options`.
-    LayerVerificationOptions default_layer_verification_option = 4;
+  // TEE platforms that the client is allowed to interact with, and can verify the evidence
+  // generated by them.
+  repeated TeePlatform supported_tee_platforms = 1;
+  // Whether to skip TEE certificate verification.
+  bool skip_tee_certificate_verification = 2;
+  // Verification option for each DICE layer.
+  map<string, LayerVerificationOptions> layer_verification_options = 3;
```

Comment:
I think it could be a `repeated` field, where each object will have a `layer_name`. So that we know the order of the layers.